api/http/routing.go

```diff
@@ -42,45 +43,45 @@ const (
 	ApiUrlManagementArtifacts           = ApiUrlManagement + "/artifacts"
 	ApiUrlManagementArtifactsList       = ApiUrlManagement + "/artifacts/list"
 	ApiUrlManagementArtifactsGenerate   = ApiUrlManagement + "/artifacts/generate"
-	ApiUrlManagementArtifactsId         = ApiUrlManagement + "/artifacts/:id"
-	ApiUrlManagementArtifactsIdDownload = ApiUrlManagement + "/artifacts/:id/download"
+	ApiUrlManagementArtifactsId         = ApiUrlManagement + "/artifacts/#id"
```

Comment:
Do we really need to move all placeholder prefix from the colon to the sharp?

Reply:
For the [go-json-rest](https://pkg.go.dev/github.com/ant0ine/go-json-rest/rest#Route) package, using colon for path parameters means that it will parse up to the next `/` or `.` in the matching string, whereas the sharp `#` will treat the `.` as part of the path parameter which is what we want in all cases anyway.

Reply:
oh, man, I didn't know it